com.bluejeans.BlueJeans.yaml

```diff
@@ -16,8 +16,9 @@ finish-args:
   - --socket=pulseaudio
   - --share=network
   - --persist=.config
-  - --persist=.macromedia
+  - --persist=.pki
   - --filesystem=xdg-videos
+  - --talk-name=org.freedesktop.Notifications
```

Comment:
Do you know what this was required by?

Reply:
Not sure, maybe desktop notifications from the BJ client?